ui/src/index.scss

```diff
@@ -13,6 +13,26 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 */
+* {
```

Comment:
Would specifying it on `body` instead of `*` work also?

Reply:
No